librz/include/rz_core.h

```diff
@@ -856,6 +804,42 @@ RZ_API RZ_BORROW const char *rz_core_analysis_name_type_to_str(RzCoreAnalysisNam
 RZ_API void rz_core_analysis_name_free(RZ_NULLABLE RzCoreAnalysisName *p);
 RZ_API RZ_OWN RzCoreAnalysisName *rz_core_analysis_name(RZ_NONNULL RzCore *core, ut64 addr);
 RZ_API bool rz_core_analysis_rename(RZ_NONNULL RzCore *core, RZ_NONNULL const char *name, ut64 addr);
+RZ_API char *rz_address_str(ut64 addr);
+
+/* cgraph.c */
+typedef enum {
```

Comment:
I recommend adding brief Doxygen description of these structures - what are they for